features/approving_a_project.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Approving a Project
+  In order for my projects to begin building support
+  As a project creator
+  I want admins to approve my project!
+
+Scenario: Admin approves a project
+  Given a project is pending
+  And I am logged in as an instance admin
+  When I approve the project
+  Then the moderator actions log shows that I approved the project
+  And the project is publicly available on the website
+  And the project is no longer available to be approved
+  And the project members receive a confirmation email
```

Comment:
I am going to pull this into another issue, since adding in the email stuff is somewhat out of scope.

Reply:
Issue created: https://github.com/wecohere/nourish.party/issues/65